db/1.9/modules/ROOT/pages/database-stored-procedure.adoc

```diff
@@ -0,0 +1,209 @@
+
+[source,xml,linenums]
+----
+<db:stored-procedure config-ref="dbConfig">
+  <db:sql>{ call doubleMyInt(:myInt) }</db:sql>
+  <db:in-out-parameters>
+      <db:in-out-parameter key="myInt" value="3"/>
+  </db:in-out-parameters>
+</db:stored-procedure>
+----
+
+== Configure the Stored Procedure with the Input and Output Parameters Fields
+
+In the Output Parameters field, specify a list of output parameters to be set on the JDBC prepared statement. Each parameter should be referenced in the SQL text using a colon prefix, for example,  `call multiply(:value, :result)`.
+
+To configure the operation and the parameter, follow the next steps:
+
+. In your Studio flow, select the *Stored procedure* operation.
+. In the operation configuration screen, set the *SQL Query Text* field to the query to execute, for example:
++
+`{ call multiplyInts(:int1, :int2, :result1, :int3, :result2) }`
++
+. Set the *Input parameters* field to `
++
+[source,xml,linenums]
+----
+{
+    'int1' : 3,
+    'int2' : 4,
+    'int3' : 5
+}
+----
++
+[start=4]
+. Set the *Output parameters* field to `Edit inline`.
+. Click the plus sign (*+*) to add a new value.
+. In the new window, set the *Key* field to `result1` and the *Type* field to `INTEGER`.
+. Repeat the previous three steps to add new values, set the *Key* field to `result2` and the *Type* field to `INTEGER`.
+. Create another value again by setting the *Key* field to `myInt` and the *Type* field to `INTEGER`.
+
+The following screenshot shows the configuration:
+
+.Stored procedure with Input and Output parameters configuration
+image::database-stored-example-3.png[Stored procedure with Input and Output parameters configuration]
+
+In the XML editor, the `<db:input-parameters>` and `<db:output-parameters>` configuration looks like this:
+
+[source,xml,linenums]
+----
+<db:stored-procedure config-ref="dbConfig">
+    <db:sql>{ call multiplyInts(:int1, :int2, :result1, :int3, :result2) }</db:sql>
+    <db:input-parameters>
+        #[{
+            'int1' : 3,
+            'int2' : 4,
+            'int3' : 5
+        }]
+    </db:input-parameters>
+    <db:output-parameters>
+        <db:output-parameter key="result1" type="INTEGER"/>
+        <db:output-parameter key="result2" type="INTEGER"/>
+        <db:output-parameter key="myInt" type="INTEGER"/>
+    </db:output-parameters>
+</db:stored-procedure>
+----
+
+== Callable Statements
+
+Database Connector supports the use of callable statements with the format:
+
+`{ call procedureName(:param1, :param2, ..., :paramN) }`
+
+The parameters matches positionally, `:paramN` matches the `Nth` parameter in the stored procedure declaration. Make sure that no whitespace is between the name of the stored procedure and the first parentheses.
+
+In the following example, you are using an Oracle database which was initialized with the creation of table named `SYSTEM.employees` and stored procedure named `createEmployee`:
```

Comment:
"In the following example, you are..." does not match the previous example intros that I have seen in the PR.
Try to standardize to the wording in the template or at least be consistent within one deliverable.

Reply:
Perhaps...:
```suggestion
The following example illustrates the Oracle database that was initialized when you created a table named `SYSTEM.employees` and stored procedure named `createEmployee`:
```